### ISSUE TITLE
bcmxcp does not disable beeper with Powerware 9125

### ISSUE THREAD

**Issue by apankevych** [networkupstools/nut#937]
'upscmd' disable command says 'OK' but it does not change the status:

`upsc pw9125
ambient.temperature: 28
ambient.temperature.high: 52
battery.charge: 100
battery.charger.status: floating
battery.energysave.delay: 5
battery.energysave.load: 10
battery.packs: 0
battery.runtime: 6238
battery.runtime.low: 2
battery.voltage:  84.1
device.mfr: Eaton
device.model: PW9125          3000i
device.part: 103002723-5501  
device.serial: RX235A0480      
device.type: ups
driver.name: bcmxcp
...
ups.beeper.status: enabled`

'mute' status can be set but beeper is still ON.

**Comment by apankevych** [comment]
System: Archlinux
community/nut 2.7.4-3
extra/linux-zen 5.9.9.zen1-1

**Comment by apankevych** [comment]
It seems it can be related to https://github.com/networkupstools/nut/issues/9.

**Comment by jimklimov** [comment]
Is this with a packaged release 2.7.4 (sadly quite aged, working on that) - not with a recent Git revision?

**Comment by apankevych** [comment]
Correct, this is with packaged release 2.7.4. I'll try with git revision as well.

**Comment by apankevych** [comment]
@jimklimov Doesn't work with recent Git revision neither(

The same behavior:
```
~> sudo upscmd -u upsuser -p password pw9125 beeper.disable
OK
~> sudo upsc pw9125 ups.beeper.status
enabled
~> sudo upscmd -u upsuser -p password pw9125 beeper.mute
OK
~> sudo upsc pw9125 ups.beeper.status
enabled
```
Version: Network UPS Tools - UPS driver controller 2.7.4-2060-gb7374ea6
